Tools-Override/tests.targets

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="GetTargetMachineInfo" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+
+  <PropertyGroup>
+    <TestRuntimeDir>$(RuntimeDir)</TestRuntimeDir>
+    <TestHostExecutable>$(TestRuntimeDir)/corerun</TestHostExecutable>
```

Comment:
>TestHostExecutable [](start = 5, length = 18)

Should we make this guy conditioned so that if it's already set we don't override it? That would be nice for testing the CLI host.